PeerioTechnologies/peerio-mobile
# fix: fixed search bar disappearing [branch ch6224]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/6224/mobile-file-search-bar-disappears-when-no-matching-files
#### Testing instructions  
1-
State: File zero state (no files and no folders)
Outcome: search bar should be hidden and zero state image should be visible ("upload something", etc...)
2-
State: Not file zero state (have files and/or folders) and search bar is empty
Outcome: File zero state image should be hidden, search bar should be visible
3-
State: Not file zero state (have files and/or folders) and search bar contains text which matches a file
Outcome: Search bar should be visible and file(s) should be visible

4- (This is the state which caused the bug previously)
State: Not file zero state and search bar contains text which doesn't match any file
Outcome: Search bar should be visible and "no files match your search" should be visible
----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
